internal/dbutil/defaults.go

```diff
@@ -3,5 +3,31 @@
 
 package dbutil
 
-// DefaultMaxIdleConns default value for database connections
-const DefaultMaxIdleConns = 100
+import (
+	"database/sql"
+	"flag"
+
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+)
+
+var (
+	maxIdleConns    = flag.Int("db.max_idle_conns", 50, "default value for database connections, -1 means the stdlib default")
+	maxOpenConns    = flag.Int("db.max_open_conns", 100, "default value for database connections, -1 means the stdlib default")
+	connMaxLifetime = flag.Duration("db.conn_max_lifetime", -1, "default value for database connections, -1 means the stdlib default")
```

Comment:
yes, i know this configuration situation sucks. i'm open to suggestions for how to easily apply configuration to all db connections. long term we should do something new with db setup, but for now this is a short term attempt at a remedy for our connection pooling woes